design/role.go

```diff
@@ -87,6 +87,21 @@ var _ = a.Resource("resource_roles", func() {
 		a.Response(d.Unauthorized, JSONAPIErrors)
 		a.Response(d.Conflict, JSONAPIErrors)
 	})
+	a.Action("hasScope", func() {
```

Comment:
You've missed the jwt protection here, @xcoulon .

Reply:
ouch! Let me fix that immediately!